eng/common/TestResources/README.md

```diff
@@ -30,7 +30,7 @@ Connect-AzAccount -Subscription 'YOUR SUBSCRIPTION ID'
 eng\common\TestResources\New-TestResources.ps1 -ServiceDirectory 'search'
 ```
 
-The `OutFile` switch would be set if you are running this for a .NET project on Windows. This will save test environment settings
+The `OutFile` switch should be set if you are running this for a .NET project on Windows. This will save test environment settings
```

Comment:
This should stay "would" or even more firmly be "will". It defaults to `$true` on Windows. "should" makes it sound like "you should set this".